### PR TITLE
Made the required changes, Thanks!

### PR DESCRIPTION
**Pull request by akshita2k**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -335,8 +336,8 @@ const data = [
         "portfolio": "http://aravindh020.github.io/"
 },
 {
-	"longitude": 76.958896,
-	"latitude": 11.000897,
+	"longitude": 77.620540,
+	"latitude": 13.079790,
```
Comment (abhinavsri360) [inline src/components/json/worldData.js:340]: Any specific reason behind changing Naveen's geolocations?

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]

File: src/components/json/worldData.js (hunk 1 above)
Comment (akshita2k) [reply src/components/json/worldData.js]: Really sorry, didn't notice that. Made another request.

File: src/components/json/worldData.js [hunk 2]
```diff
@@ -524,6 +525,22 @@ const data = [
       "twitter": "https://twitter.com/_AnmolKaur_",
       
     }
+	{
+        "longitude": 77.620540,
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:529]:
```suggestion
        longitude: 77.620540,
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@akshita2k Quotes aren't required.

**Pull request closed without merging**